massgov/mayflower
# DP-27858 a11y illustrated link label color adjustment for color contrast

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Darken the illustrated link label font color to meet the minimum required color contrast ratio.

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-27858)
- [Github issue]()

## Steps to Test


1.  Go to /patternlab/patterns/05-pages-service/05-pages-service.html
2. Check the font color of the illustrated link label against its background color in 
    - Find the one in the First time? section is rgba(20,85,143,.9)
    - Find the one in the What you need to know section is rgba(20,85,143,.8)
7. You can check their color contrast with https://contrastchecker.com/.  Get the hex code for each label color with color picker.  You cannot check color contrast with rgba format.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
